node/src/components/block_synchronizer.rs

```diff
@@ -926,13 +989,16 @@ impl<REv: ReactorEvent> Component<REv> for BlockSynchronizer {
         rng: &mut NodeRng,
         event: Self::Event,
     ) -> Effects<Self::Event> {
-        if self.paused {
-            warn!(%event, "BlockSynchronizer: not currently enabled - ignoring event");
-            return Effects::new();
-        }
-
         // MISSING EVENT: ANNOUNCEMENT OF BAD PEERS
         match (&self.status, event) {
+            (
+                ComponentStatus::Paused,
+                Event::Request(BlockSynchronizerRequest::Status { responder }),
+            ) => responder.respond(self.status()).ignore(),
+            (ComponentStatus::Paused, _) => {
+                warn!("component paused - ignoring event");
```

Comment:
this should be downgraded to debug! to avoid putting spurious WARNs into the logs when the status endpoints are queried on validators...it isn't actually problematic behavior for the synchronizer to be paused as that is part of its normal lifecycle.